toolkits/global/packages/global-status-message/README.md

```diff
@@ -12,114 +12,115 @@ The `global-status-message` component uses the `DEFAULT` branding across _all_ o
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/warning.png" />
 </p>
 
-```html
-<!-- Boxed -->
-<div class="c-status-message c-status-message--warning c-status-message--boxed">...</div>
-```
-
 <p>
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/boxed-warning.png" />
 </p>
 
-**Error**
+### Type: error
 
-```html
-<!-- Default -->
-<div class="c-status-message c-status-message--error">...</div>
+```json
+"type": "error"
 ```
+
 <p>
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/error.png" />
 </p>
 
-```html
-<!-- Boxed -->
-<div class="c-status-message c-status-message--error c-status-message--boxed">...</div>
-```
-
 <p>
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/boxed-error.png" />
 </p>
 
-**Info** 
+### Type: info
 
-```html
-<!-- Default -->
-<div class="c-status-message c-status-message--info">...</div>
+```json
+"type": "info"
 ```
 
 <p>
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/info.png" />
 </p>
 
-```html
-<!-- Boxed -->
-<div class="c-status-message c-status-message--info c-status-message--boxed">...</div>
-```
-
 <p>
     <img width="300" src="https://github.com/springernature/frontend-toolkits/blob/global-status-message/toolkits/global/packages/global-status-message/img/boxed-info.png" />
 </p>
+
+### Live region
+
+Some status messages appear with a page/screen as it is loaded. Others may appear or change subject to JavaScript events. In the latter case, the status component should use an ARIA live region. 
+
+```json
+"liveRegion": "true"
+```
+
+When used correctly, this will mean screen readers announce the status message whenever it is added or changed.
+
+#### Status changed
+
+A status change just requires you to update the `message` and (where applicable) `type` properties. This is enough to trigger announcement by the live region.
+
+#### Status added
+
+To trigger announcement of a new status message, the live region must be added to the page before the icon and message inside it.
+
+```html
+<!-- first add the wrapper -->
+<div class="c-status-message" role="status" aria-live="polite">
+</div>
+
+<!-- then add the contents -->
+<div class="c-status-message" role="status" aria-live="polite">
+    <svg class="c-status-message__icon" width="24" height="24" role="img" aria-label="warning:" focusable="false">
+        <use xlink:href="path/to/warning.svg#i-warning"></use>
+    </svg>
+	<div class="c-status-message__message" tabindex="-1"{{#if id}} id="{{id}}-message"{{/if}}>
```

Comment:
I recognise the pattern from the error-summary in regards to tabindex. So I would raise the same comment. At least so far it is consistent.